src/clr-angular/forms/styles/_mixins.forms.scss

```diff
@@ -189,3 +189,55 @@
     outline: 0;
   }
 }
+
+@mixin range-style(
+  $height,
+  $background-color,
+  $background-color-prop: null,
+  $use-custom-props: $clr-use-custom-properties
+) {
+  -webkit-appearance: none;
+  left: 0;
+  height: $height;
+
+  @if $use-custom-props != true or $background-color-prop == null {
+    background-color: $background-color;
+  } @else {
+    @include css-var(background-color, $background-color-prop, $background-color, $use-custom-props);
+  }
+}
+
+@mixin range-track-style(
+  $height,
+  $background-color,
+  $background-color-prop: null,
+  $use-custom-props: $clr-use-custom-properties
+) {
+  height: $height;
+  cursor: pointer;
+
+  @if $use-custom-props != true or $background-color-prop == null {
```

Comment:
You don't need these checks here. `@include css-var()` will do this for you. This is an internal mixin, so no one should be passing null into the `$background-color-prop` parameter.

Reply:
Done!